src/components/base/JSwitch/jSwitch.scss

```diff
@@ -0,0 +1,57 @@
+.j-switch {
+  all: initial;
+  display: inline-block;
+
+  &:not(:last-child) {
+    margin-bottom: $checkbox-item-margin-bottom;
+  }
+
+  > .field {
+    width: 42px;
+    height: 24px;
+    display: block;
+    cursor: pointer;
+    position: relative;
+
+    > .checkbox {
+      display: none;
+    }
+
+    > .switch {
+      width: 100%;
+      height: 100%;
+      display: block;
+      cursor: pointer;
+      border-radius: 100px;
+      background-color: rgba($dusk, 0.2);
+      transition: background-color 0.3s ease;
+
+      &::after {
+        content: '';
+        top: 2px;
+        left: 2px;
+        width: 20px;
+        height: 20px;
+        position: absolute;
+        border-radius: 100px;
+        background-color: #fff;
+        will-change: transform;
+        transition: transform 0.3s ease;
+      }
+    }
+
+    > .checkbox:checked ~ .switch {
```

Comment:
It's ok, but not so obviously.
For example, some developer will need to make wrapper for `.switch` (for animation or any other purposes), and behaviour of the component will be broken, if he forget to change css.
We have controlled by js prop `isChecked` so let's use it to toggle some parent class (modifier, e.g. `-checked`)
```
<div className={classNames('j-switch', isChecked && '-checked')}>
```